pkg/defaultRules/defaultRules.yaml

```diff
@@ -1869,3 +1869,58 @@ rules:
         $ref: "#"
       items:
         $ref: "#"
+  - id: 61
+    name: Ensure seccomp profile must not be explicitly set to Unconfined
+    uniqueName: CONTAINERS_INCORRECT_SECCOMP_PROFILE
+    enabledByDefault: true
+    documentationUrl: "https://www.datree.io/resources/eks-best-practices-you-should-know"
+    messageOnFailure: "Seccomp profile is explicitly set to Unconfined - unset it to retain the pod's or container's isolation"
+    category: Containers
+    schema:
```

Comment:
from the K8s docs:
`...you can configure Kubernetes to use the profile that the container runtime prefers by default, rather than falling back to Unconfined`

so, basically, not setting `seccompProfile` means you are using `Unconfined` as your seccomp profile.
also, `seccompProfile` supports only the values "Localhost", "RuntimeDefault", "Unconfined", so maybe a validation that "Localhost" or "RuntimeDefault" value is set will be more comprehensive.
WDYT?

Reply:
So should check be that the seccompProfile *does* defined in the config, and that it's only "Localhost" or "RuntimeDefault"?

Reply:
I think this should be the logic:
seccompProfile has to be set && it has to be "Localhost" or "RuntimeDefault